demo/atom/backToTop/playground

```diff
@@ -0,0 +1,108 @@
+          id="Shape"
+        />
+      </g>
+    </g>
+  </svg>
+)
+
+const Paragraph = () => (
+  <p>
+    Lorem ipsum dolor sit amet consectetur adipiscing, elit dignissim etiam
+    congue ultricies. Commodo sociis massa a potenti dictumst turpis laoreet
+    elementum, pulvinar hendrerit risus vivamus rhoncus etiam sapien congue sem,
+    luctus tristique sagittis mollis ac convallis cubilia. Conubia dictum
+    maecenas eleifend tincidunt nibh nam turpis fringilla vulputate, volutpat
+    pretium neque platea phasellus tempus interdum habitant, sociis convallis
+    taciti viverra aliquam nec metus auctor.
+  </p>
+)
+
+class ContainerDemo extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      paragraphs: 20,
+      dark: false
+    }
+  }
+  render() {
+    const {dark, text, paragraphs} = this.state
+    const customProps = {}
+    customProps.style = dark ? backToTopStyles.LIGHT : backToTopStyles.DARK
+    if (text) {
+      customProps.textTop = 'TOP'
+    }
+    return (
+      <div>
+        <div style={{display: 'flex', justifyContent: 'space-between', marginBottom: '10px'}}>
+          <button
+            onClick={() => {
+              this.setState({paragraphs: this.state.paragraphs + 20})
+            }}
+          >
+            Add 20 paragraphs
+          </button>
+        </div>
+        <div style={{display: 'flex', justifyContent: 'space-between'}}>
+          <button
+            onClick={() => {
+              this.setState({dark: false})
+            }}
+            style={ !dark ? { background: '#4d4d4d', color: 'white' } : {}}
+          >
+            BackToTop Dark Mode
+          </button>
+          <button
+            onClick={() => {
+              this.setState({dark: true})
+            }}
+            style={ dark ? { background: '#4d4d4d', color: 'white' } : {}}
+          >
+            BackToTop Light Mode
+          </button>
+          <button
+            onClick={() => {
+              this.setState({text: false})
+            }}
+            style={ !text ? { background: '#4d4d4d', color: 'white' } : {}}
+          >
+            BackToTop Without Text
+          </button>
+          <button
+            onClick={() => {
+              this.setState({text: true})
+            }}
+            style={ text ? { background: '#4d4d4d', color: 'white' } : {}}
+          >
+            BackToTop With Text
+          </button>
+        </div>
+        <div style={ dark ? { marginTop: '10px', padding: '10px', background: '#4d4d4d', color: 'white'} : {padding: '10px'}}>
+          {new Array(paragraphs)
+            .fill(0)
+            .map((elem, index) => <Paragraph key={index} />)}
+        </div>
+        <AtomBackToTop
+          iconTop={iconTop}
+          refContainer=".sui-Studio-main"
```

Comment:
the demo doesn't work in dev mode as this class doesn't exists there

Reply:
solved!